test/unit/detailed_guide_test.rb

```diff
@@ -121,4 +121,15 @@ class DetailedGuideTest < ActiveSupport::TestCase
 
     assert_equal detailed_guide.related_detailed_guide_content_ids, [some_detailed_guide.content_id]
   end
+
+  test 'related_detailed_guide_ids dedupes content_ids' do
+    some_detailed_guide = create(:detailed_guide, content_id: 'deadbeef-cafe-babe-c0ffe1')
+    some_other_detailed_guide = create(:detailed_guide, content_id: 'deadbeef-cafe-babe-c0ffe1')
```

Comment:
These two lines are wrong (`undefined content_id=`) but I'm at a loss as to how to fix them.